rest-api/bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app');
+var debug = require('debug')('example:server');
+var http = require('http');
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+var port = normalizePort(process.env.PORT || '3000');
+app.set('port', port);
+
+/**
+ * Create HTTP server.
+ */
+
+var server = http.createServer(app);
+
+/**
+ * Listen on provided port, on all network interfaces.
+ */
+
+server.listen(port);
```

Comment:
Can we simplify these 3 lines with just `app.listen(port);` pls? 
For what the example app is im not sure they are necessary (that includes using nodes `http` module)

Reply:
Wow! By adding this change, it makes my code much simpler! Thanks a lot!!!